approval_polls/templates/approval_polls/detail.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'public_base.html' %}
 
 {% block content %}
-  <h3>{{ poll.question }}</h3>
+  <h3>{{ poll.question }}&nbsp;&nbsp;&nbsp;&nbsp;{{ poll.user.username }}</h3>
```

Comment:
This will look better if put after the form and before the 'See Results' paragraph and centered.